actions.js

```diff
@@ -100,11 +102,34 @@ function setFinalImage(index, deleteImage) {
     randomImage.prop("src", IMAGES[index]);
     randomImage.addClass("random-selected");
     $("#pick-button").prop("disabled", false);
+
+    pickedImageHistory.push(IMAGES[index]);
+    updatePickedImages();
+
     if (deleteImage) {
         deleteSelectedImage(index);
     }
 }
 
+function updatePickedImages() {
+    roundsPlayed += 1;
+    if (roundsPlayed >= 1) {
+        $("#previous-image-1").prop("src", pickedImageHistory[pickedImageHistory.length - 1]).css("display", "");
+    }
+    if (roundsPlayed >= 2) {
+        $("#previous-image-2").prop("src", pickedImageHistory[pickedImageHistory.length - 2]).css("display", "");
+    }
+    if (roundsPlayed >= 3) {
+        $("#previous-image-3").prop("src", pickedImageHistory[pickedImageHistory.length - 3]).css("display", "");
+    }
```

Comment:
the variable roundsPlayed is equal to the pickedImageHistory.length. Seems like a better idea. The idea below (of having all previous images displayed), by the way, would allow to do this with a for-loop instead.